microsoft-365/enterprise/modern-custom-extensions.md

```diff
@@ -48,9 +48,10 @@ If an extension is impacting page load time or there are too many extsnions on t
 Information available in the results includes:
 
 - **Name and ID** shows identifying information that can help you find the extension on the page
-- **Total** shows the total time for the extension to initialize and load
-- **Module Load** shows the time taken to fetch and load the extension
-- **Init** shows the time taken for the extension to initialize
+- **Total** shows the total time for the extension to module load and initialize. It is the total relative time taken by the extension to execute on the page, from beginning to the end.
+- **Module Load** shows the time taken to download, evaluate and load the extensions JavaScript and CSS files. It will then start the Init process.
```

Comment:
```suggestion
- **Module Load** shows the time taken to download, evaluate, and load the extension's JavaScript and CSS files. It will then start the Init process.
```